components/raftstore/src/store/config.rs

```diff
@@ -143,7 +143,7 @@ pub struct Config {
 
     #[config(hidden)]
     pub use_delete_range: bool,
-
+    pub generate_pool_size: usize,
```

Comment:
if it can not be online changed, please specify `#[config(skip)]`.

Reply:
Doc is needed. Also for `config-template.toml`.

Reply:
DONE

Reply:
Do we have a pool that can support dynamically adjusting threads?

Reply:
snap_generating_pool_size

Reply:
@nolouch not yet. But the PR for raftstore's dynamic-sized thread pool is on the way.